cli/server/server.go

```diff
@@ -351,7 +370,15 @@ func startServer(ctx *cli.Context) error {
 	if err != nil {
 		return cli.NewExitError(fmt.Errorf("failed to create network server: %w", err), 1)
 	}
-	rpcServer := server.New(chain, cfg.ApplicationConfiguration.RPC, serv, log)
+	orc.SetOnTransaction(func(tx *transaction.Transaction) {
```

Comment:
But what's worse is that this `cli` code implements node's logic.

Reply:
Fixed.